ts/src/base/functions/crypto.ts

```diff
@@ -59,9 +59,9 @@ function ecdsa (request: Hex, secret: Hex, curve: CurveFn, prehash: CHash = null
                 //     publicKey  [1] BIT STRING OPTIONAL
                 // }
                 if (typeof asn1.sub[2].sub !== null && asn1.sub[2].sub.length > 0) {
-                    const oid = asn1.sub[2].sub[0].content (undefined);
-                    if (supportedCurve[oid] === undefined) throw new Error('Unsupported curve');
-                    curve = supportedCurve[oid];
+                    const oid = asn1.sub[2].sub[0].content (undefined);  // TODO: Argument of type 'undefined' is not assignable to parameter of type 'number'.ts(2345)
+                    if (supportedCurve[oid] === undefined) throw new Error('Unsupported curve');  // TODO: Type 'null' cannot be used as an index type.ts(2538)
+                    curve = supportedCurve[oid];  // TODO: Type 'null' cannot be used as an index type.ts(2538)
```

Comment:
@carlosmiei I didn't know what to do about these errors

Reply:
I think we should focus more on typing methods inside the transpiled code and not here

Reply:
some of these methods are very sensitive and we might be introducing regressions

Reply:
> I think we should focus more on typing methods inside the transpiled code and not here

well it will need to be updated eventually, I'm not sure what to set the types to